concent_api/core/tests/test_core_views.py

```diff
@@ -464,6 +464,7 @@ def test_send_should_return_http_400_if_task_to_compute_deadline_is_not_an_integ
             self.assertEqual(response_400.status_code, 400)
             self.assertIn('error', response_400.json().keys())
 
+    @freeze_time("2017-11-17 10:00:00")
```

Comment:
Why this changed ?

Reply:
So that all messages are created with this `timestamp` instead of previous current time.

Reply:
:+1: